src/guides/v2.3/install-gde/prereq/apache.md

```diff
@@ -71,10 +71,10 @@ The following sections discusses how to install or upgrade Apache:
 *  Install Apache
 *  Upgrade to Apache 2.4 on Ubuntu 12 to use PHP 7.3+
```

Comment:
Magento 2.3 doesn't currently support PHP 7.4. Consider whether you need to call out Ubuntu 12 at this stage of its lifecycle. If you have to keep this here, break the symlink

Reply:
Bah...this is when I thought it would be 2.4 only.

Reply:
Fixed.